src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -940,8 +943,10 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         manager.register_pass<ov::intel_gpu::SwiGLUFusion>();
         manager.register_pass<ov::intel_gpu::IndirectKVCache>();
 
-        auto kv_cache_compression_dt = config.get_property(ov::hint::kv_cache_precision);
-        manager.register_pass<ov::intel_gpu::KVCacheCompression>(kv_cache_compression_dt);
+        if (!device_info.supports_immad) {
```

Comment:
Not sure if anyone wants to try using KV-cache compression for dGPUs, but this will disable it even if the user explicitly enables it. Probably, we can limit reading/applying rt-info for systolic platforms instead?